code/modules/food_and_drinks/food/foods/frozen.dm

```diff
@@ -17,6 +17,8 @@
 	icon = 'icons/obj/food/frozen_treats.dmi'
 	icon_state = "clowncake"
 	slice_path = /obj/item/reagent_containers/food/snacks/frozen/clowncakeslice
+	slices_num = 5
+	bitesize = 3
 	list_reagents = list("nutriment" = 20, "sugar" = 5, "vitamin" = 5, "banana" = 15)
 	tastes = list("cake" = 5, "sweetness" = 2, "carrot" = 1)
```

Comment:
carrot is intended?

Reply:
...You see bananas magically morph into tasting like carrots.

Yeah I didn't notice that and it wasn't a recent change I just wanted the cake to slice lol.

Reply:
Hey, its clown magic :)